js/sticky-header.js

```diff
@@ -0,0 +1,48 @@
+$(function() {
+    // header hide and show on scroll up/down
+    var didScroll;
+    var lastScrollTop = 0;
+    var delta = 5;
+    var initialHeaderHeight = $('header.primary').outerHeight();
+
+    // set top padding on body so header doesn't overlap content
+    $('body').css('padding-top', initialHeaderHeight);
+
+    $(window).scroll(function(event){
+        didScroll = true;
+    });
+
+    setInterval(function() {
```

Comment:
I would look into using requestAnimationFrame instead of setting an interval.

http://creativejs.com/resources/requestanimationframe/
https://css-tricks.com/using-requestanimationframe/